common/config.go

```diff
@@ -0,0 +1,273 @@
+		ContentType: constants.DefaultKubeContentType,
+		QPS:         constants.DefaultKubeQPS, Burst: constants.DefaultKubeBurst, NodeUpdateFrequency: constants.DefaultKubeUpdateNodeFrequency * time.Second,
+		KubeConfig: ""}},
+		CloudHub: CloudHubSt{IPAddress: "0.0.0.0", Port: 10000, CA: "/etc/kubeedge/ca/rootCA.crt", Cert: "/etc/kubeedge/certs/edge.crt",
+			Key: "/etc/kubeedge/certs/edge.key", KeepAliveInterval: 30, WriteTimeout: 30, NodeLimit: 10},
+		DeviceController: DeviceControllerSt{Kube: KubeEdgeCntrllerConfig{Master: "http://localhost:8080", Namespace: constants.DefaultKubeNamespace,
+			ContentType: constants.DefaultKubeContentType,
+			QPS:         constants.DefaultKubeQPS, Burst: constants.DefaultKubeBurst, KubeConfig: ""}},
+	}
+	if err := Write2File(path, controllerData); err != nil {
+		return err
+	}
+	return nil
+}
+
+//WriteCloudModulesYamlFile writes modules.yaml for cloud component
+func WriteCloudModulesYamlFile(path string) error {
+	modulesData := ModulesYaml{Modules: ModulesSt{Enabled: []string{"devicecontroller", "controller", "cloudhub"}}}
+	if err := Write2File(path, modulesData); err != nil {
+		return err
+	}
+	return nil
+}
+
+//WriteCloudLoggingYamlFile writes logging yaml for cloud component
+func WriteCloudLoggingYamlFile(path string) error {
+	loggingData := LoggingYaml{LoggerLevel: "INFO", EnableRsysLog: false, LogFormatText: true, Writers: []string{"file", "stdout"}, LoggerFile: "edgecontroller.log"}
+	if err := Write2File(path, loggingData); err != nil {
+		return err
+	}
+	return nil
+}
+
+//WriteEdgeLoggingYamlFile writes logging yaml for edge component
+func WriteEdgeLoggingYamlFile(path string) error {
+	loggingData := LoggingYaml{LoggerLevel: "DEBUG", EnableRsysLog: false, LogFormatText: true, Writers: []string{"stdout"}}
+	if err := Write2File(path, loggingData); err != nil {
+		return err
+	}
+	return nil
+}
+
+//WriteEdgeModulesYamlFile writes modules.yaml for edge component
+func WriteEdgeModulesYamlFile(path string) error {
+	modulesData := ModulesYaml{Modules: ModulesSt{Enabled: []string{"eventbus", "servicebus", "websocket", "metaManager", "edged", "twin"}}}
+	if err := Write2File(path, modulesData); err != nil {
+		return err
+	}
+	return nil
+}
+
+//MQTTMode = # 0: internal mqtt broker enable only. 1: internal and external mqtt broker enable. 2: external mqtt broker enable only.
+type MQTTMode uint8
+
+//Different message exchange mode supported in KubeEdge using MQTT
+const (
+	MQTTInternalMode MQTTMode = iota
+	MQTTInternalExternalMode
+	MQTTExternalMode
+)
+
+//MQTTQoSType = # 0: QOSAtMostOnce, 1: QOSAtLeastOnce, 2: QOSExactlyOnce.
+type MQTTQoSType uint8
+
+//Different MQTT QoS
+const (
+	MQTTQoSAtMostOnce MQTTQoSType = iota
+	MQTTQoSAtLeastOnce
+	MQTTQoSExactlyOnce
+)
+
+//MQTTSt contains MQTT specific config to use MQTT broker
+type MQTTSt struct {
```

Comment:
Can we remove St from the end of the structure name ?
Applicable to all similar places

Reply:
there should be some difference between the actual structure type an the variable/field name. Hence "St --> Structure" is added. It helps in code readability, specially developers who are not using advanced IDE's like Goland or VScode. Also when you search for strings in a repo or folders it is always unique.

Reply:
Can we change to MQTTConfig ? suffixing with the datatype initials is not looking good

Reply:
Done